bin/sozo/src/utils.rs

```diff
@@ -0,0 +1,37 @@
+use anyhow::Result;
+use dojo_world::utils::{execution_status_from_maybe_pending_receipt, TransactionWaiter};
+use starknet::core::types::{ExecutionResult, InvokeTransactionResult};
+use starknet::providers::Provider;
+
+pub async fn handle_transaction_result<P>(
+    provider: P,
+    transaction_result: InvokeTransactionResult,
+    wait_for_tx: bool,
+    show_receipt: bool,
+) -> Result<()>
+where
+    P: Provider + Send,
+{
+    println!("\nTransaction hash: {:#x}", transaction_result.transaction_hash);
```

Comment:
```suggestion
    println!("Transaction hash: {:#x}", transaction_result.transaction_hash);
```
lets remove the newline